### PR TITLE
LPS-192269 System fields searchable configuration behavior

### PR DESCRIPTION
**Pull request by Artartur**
Related Issue: [LPS-192269](https://liferay.atlassian.net/browse/LPS-192269)

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/SearchableContainer.tsx [hunk 1]
```diff
@@ -59,11 +59,14 @@ export function SearchableContainer({
...
-					disabled={objectField.businessType === 'Encrypted'}
+					disabled={
+						objectField.businessType === 'Encrypted' ||
+						objectField.system
+					}
 					label={Liferay.Language.get('searchable')}
 					name="indexed"
 					onToggle={(indexed) => setValues({indexed})}
-					toggled={objectField.indexed}
+					toggled={objectField.indexed || objectField.system}
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/SearchableContainer.tsx:69]: @Artartur, the `indexed` property will define whether the object is searchable or not, there is no point in forcing it to be `toggled` if it is `system` only in the frontend if it is not passed to the backend. I confirmed with the backend folks here, and the only `system` field that makes sense to be seachebe is the `id` field, so the backend is right, the only thing we need to do in the frontend is disable the toggle when the field is system, you You can remove changes from this line.

**Review by danvb112 — CHANGES_REQUESTED** [review]
@Artartur, good job on this PR, I have only one comment about it! :smile:

**Comment by Artartur** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/SearchableContainer.tsx (hunk 1 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/SearchableContainer.tsx]: Sure, I got it, thx for explain, done!

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
Sent to: https://github.com/brianchandotcom/liferay-portal/pull/140805

**Pull request closed without merging**